editor-components/team/team-7/team7.tsx

```diff
@@ -3,310 +3,326 @@ import styles from "./team7.module.scss";
-                                        <div className={this.decorateCSS("person-info")}>
-                                            <div className={this.decorateCSS("text-group")}>
-                                                <h1 className={this.decorateCSS("item-name")}>{item.name}</h1>
-                                                <p className={this.decorateCSS("item-position")}>{item.position}</p>
-                                            </div>
-                                            <div className={this.decorateCSS("icons-bar")}>
-                                            {item.icons && item.icons.map(
-                                           (card: Icons, indexIcons: number) => (
-                                          <ComposerLink
-                                            key={indexIcons}
-                                            path={card.url}
-                                         >
-                                            <ComposerIcon name={card.icon} 
-                                            propsIcon={{
-                                                className:this.decorateCSS("icon")
-                                            }}/>
-                                         </ComposerLink>
-                                            )
-                                            )}
-                                            </div>
-                                        </div>
-                                    </div>
-                                )
-                            })}
+    this.addProp({
+      type: "number",
+      key: "itemCount",
+      displayer: "Item Count",
+      value: 4,
+      max: 5,
+    });
+  }
+
+  getName(): string {
+    return "Team 7";
+  }
+
+  render() {
+    const badge = this.getPropValue("badge");
+    const title = this.getPropValue("title");
+    const description = this.getPropValue("description");
+
+    const badgeValue = badge.props.html;
+    const titleValue = title.props.html;
+    const descriptionValue = this.castToString(description);
+
+    const contentAligment = Base.getContentAlignment();
+
+    console.log(contentAligment, "contentAligment");
+
+    return (
+      <Base.Container className={this.decorateCSS("container")}>
+        <Base.MaxContent className={this.decorateCSS("max-content")}>
+          <Base.VerticalContent className={this.decorateCSS("basic-page")}>
+            {contentAligment === "left" ? (
+              <Base.VerticalContent className={this.decorateCSS("up-page")}>
+                {badgeValue && <Base.SectionSubTitle className={this.decorateCSS("badge")}>{this.getPropValue("badge")}</Base.SectionSubTitle>}
+                <div className={this.decorateCSS("up-page-bottom")}>
+                  {titleValue && <Base.SectionTitle className={this.decorateCSS("title")}>{this.getPropValue("title")}</Base.SectionTitle>}
```

Comment:
title value will always render cause the system returns a HTML tag. check others